src/components/oscillator/test/oscillator_tests-implementation.adb

```diff
@@ -64,6 +66,18 @@ package body Oscillator_Tests.Implementation is
 
       -- Make sure parameters changed:
       Check_Parameters (2.9, 7.8, 0.4);
+
+      -- Validate a parameter:
+      Test_Param_Update.Operation := Parameter_Enums.Parameter_Operation_Type.Validate;
```

Comment:
New test for this

Reply:
Moved to new test in [3eafa26](https://github.com/lasp/adamant_example/pull/31/commits/3eafa26fb710bde3d62750f3e401090cff24b15c).